packages/jest-circus/src/utils.js

```diff
@@ -220,7 +220,7 @@ const _formatError = (error: ?Exception): string => {
   } else if (error.message) {
     return error.message;
   } else {
-    return String(error);
+    return `${String(error)} thrown`;
```

Comment:
this is the way jasmine throws errors now, i just change it to this to be compatible with current snapshots